backend/bom-bom-server/src/main/java/me/bombom/api/v1/pet/domain/Pet.java

```diff
@@ -0,0 +1,48 @@
+package me.bombom.api.v1.pet.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import me.bombom.api.v1.common.BaseEntity;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Pet extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private Long memberId;
+
+    @Column(nullable = false)
+    private Long stageId;
+
+    private int currentScore = 0;
```

Comment:
p2

`@Column`은 DB에 대한 설정이니 DB에서는 int타입이라도 null이 들어갈 수 있기 때문에 nullable로 해주는게 좋을 것 같아요 
또한 DB가 항상 안전한 초기값을 갖도록 `columnDefinition`도 같이 설정해주면 좋을 것 같아요 😊

```suggestion
    @Column(nullable = false, columnDefinition = "int default 0")
    private int currentScore = 0;
```

Reply:
헉 아까 `nullable = false` 옵션 추가하는거 피드백 받았는데 여긴 놓쳤네요..!!
반영하겠습니다! 캐치 감사해요 :)